### DISCUSSION TITLE
Not able to use variable declared in less file

### DISCUSSION THREAD

**Discussion by HenriqueAurelio** [less/less.js#3689] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31359644/151277495-5a3f95d0-c723-4944-b72a-c8731306bbf1.png)
![image](https://user-images.githubusercontent.com/31359644/151277534-f3437ec6-e79f-449d-9a63-d71f6a502444.png)
![image](https://user-images.githubusercontent.com/31359644/151277562-8aa5d7a7-1e9c-4d42-831a-56d7d30630a3.png)

Anyone knows the problem?

**Comment by matthew-dean** [comment, marked as answer]
In a Vue file, when you add a pre-processor, each `<style>` block is compiled independently. Same with a `.less` import into the `<script>` block. These are two separate rendering processes which don't know about each other.

So, you need to move this to a native Less `@import` in the `<style>` block. Some loaders also allow you to "auto-inject" an import at the head of the `<style>` block, I think.

**Reply by HenriqueAurelio** [reply]
thanks a lot dude :)
